### ISSUE TITLE
[event based orderbook] Snapshot to disk

### ISSUE THREAD

**Issue by e00E** [gnosis/dex-services#793]
Currently we fetch all past events on start up.
It would be better to store a snapshot on disk and start from that to speed this up.

I would base this on https://github.com/gnosis/dex-services/pull/795 .

* In UpdatingOrderbook we would in `new` try to read a serialized `Orderbook` from disk
* from the orderbook get the last handled `block_number` (need to add a new method for this)
* continue as usual in `UpdatingOrderbook::update_with_events_forever` but don't start from `BlockNumber::Earliest`, instead use `last_handled_block - BLOCK_RANGE`

**Comment by e00E** [comment]
For getting the last handled block number after restoring from disk, I found this snippet in a branch where I had already implemented this before:

```rust
pub fn last_event_block_number(&self) -> Option<u64> {
    Some(self.events.iter().next_back()?.0.block_number)
}
```

This would  go into `Orderbook.rs`.

**Comment by bh2smith** [comment]
closed by #805 and #809
